content/300-guides/125-development-environment/100-editor-setup.mdx

```diff
@@ -40,7 +40,8 @@ You can install the official [Prisma VS Code extension](https://marketplace.visu
 
 ### Sublime Text
 
-- [sublimeText3](https://github.com/prisma/sublimeText3) provides syntax highlighting of the Prisma Schema Language.
+- [Prisma](https://packagecontrol.io/packages/Prisma) - For Sublime Text 3 & 4 - Provides syntax highlighting for the Prisma Schema Language. 
+- [LSP-prisma](https://packagecontrol.io/packages/LSP-prisma) - For Sublime Text 4 - Language Server helper package for Prisma schema files that uses Prisma's Language Server to provide linting, error checking, formatting, autocompletion, renaming etc. Note: It requires the Prisma package to be installed.
```

Comment:
How about we add a GH link as well for the source code?

```suggestion
- [Prisma](https://packagecontrol.io/packages/Prisma) - For Sublime Text 3 & 4 - Provides syntax highlighting for the Prisma Schema Language. ([Source Code](https://github.com/Sublime-Instincts/PrismaHighlight/))
- [LSP-prisma](https://packagecontrol.io/packages/LSP-prisma) - For Sublime Text 4 - Language Server helper package for Prisma schema files that uses Prisma's Language Server to provide linting, error checking, formatting, autocompletion, renaming etc. Note: It requires the Prisma package to be installed. ([Source Code](https://github.com/Sublime-Instincts/LSP-prisma))
```

Reply:
Good idea 👍🏼